contracts/rateproducer/include/rateproducer.hpp

```diff
@@ -43,7 +44,7 @@ using eosio::public_key;
 */
 namespace eosio {
     constexpr name system_account{"eosio"_n};
-    constexpr name eden_account{"genesis.eden"_n};
+    constexpr name eden_account{"genesisdeden"_n};
```

Comment:
I'm going to leave it in this way, later, when this PR is merge, I'm going to create another one with the `genesis.eden` account and with its corresponding .abi and .wasm files